lib/assessment.rb

```diff
@@ -20,21 +23,24 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    if numbers == odd?
+      return true
   end
+    end
+
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    my_array.select{|item| item%2==0 }
```

Comment:
To check if every element matches a criterion, use `all?`
`numbers.all?(&:even?)`
Or, alternatively
`numbers.all? {|num| num.even?}`